### ISSUE TITLE
The broken line is blocked

### ISSUE THREAD

**Issue by wjhcc2018** [highcharts/highcharts#13757]
#### Expected behaviour
Polylines can be displayed completely
#### Actual behaviour
![image](https://user-images.githubusercontent.com/6789138/85353380-ab023280-b53a-11ea-8e09-37693f58bda7.png)
#### Live demo with steps to reproduce

https://jshare.com.cn/temp/tRG0wl

**Comment by pawelfus** [comment]
Hi @wjhcc2018 

Thank you for reporting the issue, but this is expected behaviour when [`plotOptions.series.clip`](https://api.highcharts.com/highcharts/plotOptions.series.clip) is set to false.

**Comment by wjhcc2018** [comment]
@pawelfus but when clip:true, the charts render wrong.
![image](https://user-images.githubusercontent.com/6789138/85498442-4914fc00-b612-11ea-8b2c-651c6359e89c.png),
you can see the lineWidth not the same

**Comment by pawelfus** [comment]
That's correct @wjhcc2018, you can either enable or disable the clip. Lines are clipped to the plotting area or not. 

You you can try manually change clip's height, for example `chart.clipRect` is used after animation to clip the series: https://jsfiddle.net/BlackLabel/ubzqjed2/
